### ISSUE TITLE
Error in the docs (docstring) of torch.nn.MultiMarginLoss

### ISSUE THREAD

**Issue by wenh06** [pytorch/pytorch#84512]
### 📚 The doc issue

In https://pytorch.org/docs/stable/generated/torch.nn.MultiMarginLoss.html, the index range should be

$i \in \left\\{ 0, \\; \cdots , \\; \text{x.size}(0) - 1 \right\\}$

rather than

$x \in \left\\{ 0, \\; \cdots , \\; \text{x.size}(0) - 1 \right\\}$.

### Suggest a potential alternative/fix

Corresponding corrections in the [docstring of `torch.nn.MultiMarginLoss`](https://github.com/pytorch/pytorch/blob/master/torch/nn/modules/loss.py#L1348) should be made as

```where :math:`i \in \left\{0, \; \cdots , \; \text{x.size}(0) - 1\right\}```

cc @svekars @holly1238 @albanD @mruberry @jbschlosser @walterddr @kshitij12345 @saketh-are
